Detectors/CPV/workflow/src/RawToDigitConverterSpec.cxx

```diff
@@ -81,6 +90,28 @@ void RawToDigitConverterSpec::run(framework::ProcessingContext& ctx)
     }
   }
 
+  if (!mPedestals && !mIsPedestalData) {
+    if (o2::cpv::CPVSimParams::Instance().mCCDBPath.compare("localtest") == 0) {
+      mPedestals = std::make_unique<o2::cpv::Pedestals>(1); // test default calibration
+      LOG(INFO) << "No reading calibration from ccdb requested, set default";
+    } else {
+      LOG(INFO) << "Getting calibration object from ccdb";
+      //TODO: configuring ccdb address from config file, readign proper calibration/BadMap and updateing if necessary
+      o2::ccdb::CcdbApi ccdb;
+      std::map<std::string, std::string> metadata;
+      ccdb.init("http://ccdb-test.cern.ch:8080"); // or http://localhost:8080 for a local installation
```

Comment:
here too.

Reply:
Dear @shahor02, can you advice a way to make it configurable? You mean to read it in init() function from framework::InitContext& ctx?
I ask because I can do something which no one will like to to my incomplete understanding.

Reply:
@sevdokim, just define e.g. mCCDBUrl data member which can be set via workflow option (with proper default, e.g. "http://ccdb-test.cern.ch:8080"). For the reference, look on https://github.com/AliceO2Group/AliceO2/blob/b80c66b08e5aca3eafcf8e09158b0f588a3bf388/Detectors/ZDC/workflow/src/ZDCDataReaderDPLSpec.cxx#L67 called from https://github.com/AliceO2Group/AliceO2/blob/b80c66b08e5aca3eafcf8e09158b0f588a3bf388/Detectors/ZDC/workflow/src/o2-zdc-raw2digits.cxx#L44-L54.

For the https://github.com/AliceO2Group/AliceO2/blob/3e937b05cd6a7e3d36989e3286d602dc6948c9a2/Detectors/CPV/simulation/src/RawWriter.cxx#L71 it will be slightly different, since the digit2raw is not a workflow but simple executable. See e.g. https://github.com/AliceO2Group/AliceO2/blob/b80c66b08e5aca3eafcf8e09158b0f588a3bf388/Detectors/ZDC/simulation/src/digi2raw.cxx#L60 for the usage of configurable ccdb, though ZDC sets its up in the `digi2raw.cxx` itself, you will need to propagate the option to `RawWriter.cxx`.

BTW, for the ccdb reading it is preferable to use the BasicCCDBManager (wrapper around the CcdbApi) since it supports caching.